Game/Resources/VoltageScene/VoltageScene.json

```diff
@@ -2774,24 +2774,24 @@
 					"SpotLight" : 
 					{
 						"ambientCoefficient" : 0.0,
-						"attenuation" : 1.0,
+						"attenuation" : 200.0,
```

Comment:
is this number correct?

Reply:
Yes, a higher value causes light to dissipate more quickly. It's used to make the spotlights beneath the fuses not leak as much behind the electric box.